js/openTrailFeature.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
+
+var openTrailFeature = (function() {
```

Comment:
I could be missing something, but I think part of the benefit of using a build tool that understands modules and transforms modules to closures under the hood is you no longer have to use this anonymous closure pattern in your code.

Since webpack converts each module file to a closure during the build step, can we get rid of these explicit closures?